internal/cmd/local/command_kafka_start.go

```diff
@@ -58,31 +65,35 @@ func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
 	}
 	defer out.Close()
 
-	buf := new(strings.Builder)
-	_, err = io.Copy(buf, out)
-	if err != nil {
-		return err
-	}
+	scanner := bufio.NewScanner(out)
+	for scanner.Scan() {
+		var response ImagePullResponse
+		text := scanner.Text()
 
-	for _, ss := range strings.Split(buf.String(), "\n") {
-		var output imagePullOut
-		if err := json.Unmarshal([]byte(ss), &output); err != nil {
-			continue
+		err := json.Unmarshal([]byte(text), &response)
+		if err != nil {
+			return err
 		}
+
 		var inBlacklist bool
 		for _, s := range statusBlacklist {
-			if output.Status == s {
+			if response.Status == s {
 				inBlacklist = true
 			}
 		}
+
 		if !inBlacklist {
-			fmt.Printf("%v\n", output.Status)
+			fmt.Printf("%v\n", response)
```

Comment:
I'm pretty sure this is the same thing
```suggestion
			fmt.Println(response)
```

Reply:
Maybe also `output` instead of `fmt`.

Reply:
Maybe also `output` instead of `fmt`.

(sorry if you get pinged again; I rewrote this comment)